starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:700|Roboto:100,300,700');
+
+#first{
```

Comment:
You were close! 

to get the image to take up the entire width you need to remove the margin, and set the width to 100% 

```
#first{
    background-image: url("../images/apple-pie.jpg"); 
    background-size: cover;
    width: 100%;
    align-content: center;
    justify-content: center;
}
```